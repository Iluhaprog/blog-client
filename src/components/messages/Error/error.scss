@import '../../../vars.scss';

.error-box {
    width: 100%;
    padding: 10px;
    background-color: $error-bg;
    border-radius: 5px;
    border: 1px solid $error;
    box-shadow: $shadow-2;

    &__message {
        margin: 0;
        color: $error;
    }
}