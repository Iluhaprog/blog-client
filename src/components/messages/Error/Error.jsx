import React from 'react';
import './error.scss';

export default props => (
    <div className='error-box'>
        <p className='error-box__message'>
            {props.message}
        </p>
    </div>
);