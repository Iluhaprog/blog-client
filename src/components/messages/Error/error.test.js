import React from 'react';
import { render } from '@testing-library/react';
import Error from './Error';

test('Test view of error message', () => {
    const { asFragment } = render(<Error message='test' />);
    expect(asFragment()).toMatchSnapshot();
})