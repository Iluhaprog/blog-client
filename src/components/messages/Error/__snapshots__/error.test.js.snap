// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test view of error message 1`] = `
<DocumentFragment>
  <div
    class="error-box"
  >
    <p
      class="error-box__message"
    >
      test
    </p>
  </div>
</DocumentFragment>
`;
